.github/workflows/main-workflow.yml

```diff
@@ -13,14 +13,11 @@ jobs:
     - name: tshirts fail
       run: |
         g++ -Wall -Wextra -Werror tshirts.cpp -o tshirts.out
-        bash .github/workflows/expectfail.sh ./tshirts.out

```

Comment:
The lines removed here will run the code. Why have they been removed?
Run the program before we can review...

Reply:
To complete this task https://github.com/clean-s-1/assignments/blob/main/pass.md 

this is what mentioned 
In your build-workflow (.github/workflows/main-workflow.yml), remove bash .github/workflows/expectfail.sh in three places - otherwise passing tests will fail the Action. Keep the rest of the line and maintain the same indentation as the previous line in the yml.